src/main/java/hudson/plugins/sonar/utils/SonarMaven.java

```diff
@@ -112,7 +112,10 @@ protected void wrapUpArguments(ArgumentListBuilder args, String normalizedTarget
 
     argsBuilder.append("sonar.branch", publisher.getBranch());
 
-    if (StringUtils.isNotBlank(getInstallation().getSonarLogin())) {
+    if(StringUtils.isNotBlank(getInstallation().getServerAuthenticationToken())) {
+      argsBuilder.appendMasked("sonar.login", getInstallation().getServerAuthenticationToken());
+    }
+    else if (StringUtils.isNotBlank(getInstallation().getSonarLogin())) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this "else" on the same line that the previous closing curly brace. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3ARightCurlyBraceSameLineAsNextBlockCheck)